.github/workflows/tests.yml

```
@@ -0,0 +1,29 @@
+name: tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  tests:
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        os: ['ubuntu-20.04', 'macos-latest']
```

Can we also add windows? 
```suggestion
        os: [ubuntu-latest, windows-latest, macos-latest]
```

---

Done, just waiting on tests to rerun